pkg/fetch/vendir.go

```diff
@@ -21,7 +22,15 @@ import (
 	kyaml "sigs.k8s.io/yaml"
 )
 
+// SourceType to extract host
+type SourceType string
+
 const (
+	// GitURL source type to extract host
+	GitURL SourceType = "gitURL"
```

Comment:
I don't think we are going to use these string values, so we could probably just switch to int (Iota).